Version 0.4.10 Source code still tries to run git when building

After downloading the source code: https://github.com/freeorion/freeorion/releases/download/v0.4.10/FreeOrion_v0.4.10_2020-07-10.f3d403e_Source.tar.gz

The `cmake/make_versioncpp.py` utility still attempts to run git to determine the version, even though the `.git` tree is not present in the above tarball.

Perhaps `cmake/make_versioncpp.py` should be rewritten to only try to run git if a `.git` directory exists, otherwise just use the release version?

Here's what the error looks like (I'm trying to compile in OpenBSD):
```
[48/270] cd /usr/ports/pobj/freeorion-0.4.10/build-amd64 && /usr/local/bin/python3.8 /usr/ports/pobj/freeorion-0.4.10/src-tarball/cmake/make_versioncpp.py /usr/ports/pobj/freeorion-0.4.10/src-tarball CMake
FAILED: CMakeFiles/freeorionversion 
cd /usr/ports/pobj/freeorion-0.4.10/build-amd64 && /usr/local/bin/python3.8 /usr/ports/pobj/freeorion-0.4.10/src-tarball/cmake/make_versioncpp.py /usr/ports/pobj/freeorion-0.4.10/src-tarball CMake
fatal: not a git repository (or any parent up to mount point /usr/ports)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Traceback (most recent call last):
  File "/usr/ports/pobj/freeorion-0.4.10/src-tarball/cmake/make_versioncpp.py", line 131, in <module>
    branch = check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD'], universal_newlines=True).strip()
  File "/usr/local/lib/python3.8/subprocess.py", line 411, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/local/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['git', 'rev-parse', '--abbrev-ref', 'HEAD']' returned non-zero exit status 128.
ninja: build stopped: subcommand failed.
*** Error 1 in . (/usr/ports/devel/cmake/cmake.port.mk:34 'do-build': @cd /usr/ports/pobj/freeorion-0.4.10/build-amd64 && exec /usr/bin/env ...)
*** Error 2 in . (/usr/ports/infrastructure/mk/bsd.port.mk:2921 '/usr/ports/pobj/freeorion-0.4.10/build-amd64/.build_done': @cd /usr/ports/g...)
*** Error 2 in /usr/ports/games/freeorion (/usr/ports/infrastructure/mk/bsd.port.mk:2584 'all': @lock=freeorion-0.4.10;  export _LOCKS_HELD=...)
```

--- a/cmake/make_versioncpp.py
+++ b/cmake/make_versioncpp.py
@@ -6,7 +6,7 @@ import sys
 import os
 from string import Template
 from datetime import datetime
-from subprocess import check_output
+from subprocess import check_output, CalledProcessError
 from platform import system
 from glob import glob
 
@@ -136,7 +136,7 @@ try:
     commit = check_output(["git", "show", "--no-show-signature", "-s", "--format=%h", "--abbrev=7", "HEAD"], universal_newlines=True).strip()
     timestamp = float(check_output(["git", "show", "--no-show-signature", "-s", "--format=%ct", "HEAD"], universal_newlines=True).strip())
     build_no = ".".join([datetime.utcfromtimestamp(timestamp).strftime("%Y-%m-%d"), commit])
-except IOError:
+except (IOError, CalledProcessError):
     print("WARNING: git not installed or not setup correctly")
 
 for generator in generators:

